Dockerfile

```diff
@@ -19,7 +19,7 @@ ENV SBT_VERSION 0.13.17
 
 # Install SBT
 RUN cd /usr/local && \
-    wget https://dl.bintray.com/sbt/native-packages/sbt/${SBT_VERSION}/sbt-${SBT_VERSION}.tgz && \
+    wget https://piccolo.link/sbt-${SBT_VERSION}.tgz && \
```

Comment:
Not sure I understand this line

Reply:
The sbt zip file no longer published to bintray anymore, that is why you have to download from `piccolo.link`.

It happens since `0.13.16`, you can cross check between the following two links:
- https://dl.bintray.com/sbt/native-packages/sbt/
- https://www.scala-sbt.org/download.html

That is why I linked the above issue as reference.

Reply:
Got it. This looks alright then